notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Configuration.scala

```diff
@@ -104,7 +107,8 @@ object Configuration {
         debug = config.getBoolean("fcm.debug"),
         dryRun = config.getBoolean("dryrun")
       ),
-      config.getInt("fcm.threadPoolSize")
+      config.getInt("fcm.threadPoolSize"),
+      config.getStringList("fcm.allowedTopicsForBatchSend").asScala.toList
```

Comment:
This new parameter will need to be added for every android worker in each environment. We need to define the type as a "StringList".

Suggest we start by sending only to breaking/international for now? Plus maybe sport?

Reply:
`breaking/international` has around 550,000 recipients so it may be a good choice.  But not many alerts were sent to this topic only (we got 3 breakers in the last 10 days) so we may need to wait

Reply:
Yes, we may add sport as well.  `breaking/uk-sport` has 10,000 recipients, and `breaking/international-sport` has around 5,000.  They occurs quite frequently - at least once every day in the past 10 days except 18-Oct.  
Maybe we add one of them

Reply:
Thanks Wai Sing :) Maybe we start with 2 allowed topics? 

For ref: I couldn't get the StringList parameter to parse correctly, for this temporary experiment I've decided to go with a simple string that I split using regex `","`

Reply:
I agree to allow two topics, maybe `breaking/international` and `breaking/international-sport`?

It sounds good to me to use a simple string with comma delimiter for a temporary experiment